central/complianceoperator/v2/checkresults/datastore/index/index.go

```diff
@@ -0,0 +1,12 @@
+package index
```

Comment:
I'm not sure this file is necessary since the `searcher` interface defines a `Count`